api/guides/delegate_descriptor_ids.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Delegate Descriptor IDs
+---
+
+# Understanding Delegate Descriptor IDs 
+
+When a data element, condition, event, or action is associated with an extension package, that resource is considered a "delegate" of that package. Each delegate within an extension package is given a unique delegate descriptor ID. The delegate descriptor ID for a particular resource allows Adobe Experience Platform Launch to determine what kind of resource it is and which extension package it belongs to.
+
+A delegate descriptor ID consists of three strings joined by double-colon characters (`::`), representing the extension package name, the delegate type, and the delegate name, respectively.
+
```

Comment:
You may want to note that the descriptor ID structure is composed to make it human readable, but the value of the id is generated and assigned by Launch during Extension Package ingestions.